docker/compose/docker-compose.yml

```
@@ -76,4 +76,5 @@ services:
 
 networks:
   devdocs-network:
-    driver: bridge
\ No newline at end of file
+    driver: bridge
+    name: devdocs-network
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)